src/state/stake/stake-constants.ts

```diff
@@ -369,7 +370,18 @@ const AURORA_POOLS: StakingTri[] = [
     allocPoint: 1,
     noTriRewards: true,
     doubleRewardToken: GBA[ChainId.AURORA]
+  }),
+  createMCV2Pool({
+    ID: 23,
+    poolId: 16,
+    tokens: [USDT[ChainId.AURORA], AUSDO[ChainId.AURORA]],
+    lpAddress: '0x6277f94a69Df5df0Bc58b25917B9ECEFBf1b846A',
+    rewarderAddress: '0x170431D69544a1BC97855C6564E8460d39508844',
+    allocPoint: 1,
+    noTriRewards: true,
+    doubleRewardToken: WNEAR[ChainId.AURORA]
```

Comment:
You can add in the instaging thing in this object.  
```suggestion
    noTriRewards: true,
    doubleRewardToken: WNEAR[ChainId.AURORA],
    inStaging: true
```